tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -609,15 +844,22 @@ def _ParseTypeString(type_str: syaml.YAML) -> Tuple[str, str]:
   type_parse = type_str.split('/')
 
   if len(type_parse) == 1:
-    print('Type improperly formatted, a namespace is missing: ', type_str)
-    raise TypeError(
-        f'Type improperly formatted, a namespace is missing: {type_str}\n' +
-        'Proper formatting is: NAMESPACE/TYPE_NAME')
+    print('[ERROR]\tNamespace is missing for type: {type_str}. Proper '
+          'format is NAMESPACE/TYPE_NAME.'.format(type_str=type_str)
+          )
+    raise TypeError('Namespace is missing for type: {type_str}. Proper '
```

Comment:
raising here will stop the validation, is this what you want?

Reply:
+1